### PR TITLE
feat: Added product listing page

### PR DESCRIPTION
**Pull request by JantuDeb**
### Product Listing

1. Filter product
2. product list
image

### CODE REVIEW

**Review by Omkar-Chaskar — APPROVED** [review]

File: pages/css/product.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.breadcrumbs {
+  padding: 0rem 0.5rem;
+}
+
+.sidebar-filter{
+    flex-shrink: 0;
+    width: 200px;
```
Comment (rkapoor10) [inline pages/css/product.css:7]: give in rems

File: pages/html/products.html [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+</head>
+
+<body>
+    <!-- HEADER -->
+    <header>
+        <nav class="nav-bar bg-white text-dark shadow-gray">
+            <div class="flex wrap between">
+                <div class="flex brand">
+                    <a href="../../index.html">
+                        <img src="../../images/hyperopia.png" alt="" class="nav-logo" />
```
Comment (rkapoor10) [inline pages/html/products.html:21]: give alt property

File: pages/html/products.html [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+                        <label>
+                            <input type="checkbox">
+                            <span>Rs.3000 - Rs.3999</span>
+                        </label>
+                    </li>
+                </ul>
+            </div>
+
+            <div>
+                <p class="text-gray h5">Ratings</p>
```
Comment (rkapoor10) [inline pages/html/products.html:193]: why h5 as class ?

File: pages/html/products.html [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+                        </h6>
+                    </div>
+                </a>
+                <a href="#" class="card flex-col radius-md">
+                    <div class="flex between">
+                        <div class="text-white bg-red p-2 px-2">Trending</div>
+                        <img src="../../images/wishlist.png" alt="wishlist" class="p-1" />
+                    </div>
+                    <div class="card-info">
+                        <img src="../../images/trending-2.webp" class="img-fluid radius-md" alt="" srcset="" />
```
Comment (rkapoor10) [inline pages/html/products.html:307]: here also give alt and remove srcset if not used

File: pages/html/products.html [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+                    </div>
+                </a>
+
+                <a href="#" class="card flex-col radius-md">
+                    <div class="flex between">
+                        <div class="text-white bg-red p-2 px-2">Trending</div>
+                        <img src="../../images/wishlist.png" alt="wishlist" class="p-1" />
+                    </div>
+                    <div class="card-info">
+                        <img src="../../images/trending-3.jpg" class="img-fluid radius-md" alt="" srcset="" />
```
Comment (rkapoor10) [inline pages/html/products.html:323]: same as above

File: pages/html/products.html [hunk 6]
```diff
@@ -0,0 +1,422 @@
...
+                        </h6>
+                    </div>
+                </a>
+                <a href="#" class="card flex-col radius-md">
+                    <div class="flex between">
+                        <div class="text-white bg-red p-2 px-2">Trending</div>
+                        <img src="../../images/wishlist.png" alt="wishlist" class="p-1" />
+                    </div>
+                    <div class="card-info">
+                        <img src="../../images/trending-4.jpg" class="img-fluid radius-md" alt="" srcset="" />
```
Comment (rkapoor10) [inline pages/html/products.html:338]: same as above

**Review by rkapoor10 — APPROVED** [review]
Code is structured. Looks good to merge.

File: pages/html/products.html (hunk 3 above)
Comment (JantuDeb) [reply pages/html/products.html:193]: I want to apply the style of h5

**Pull request merged**